packages/happy-dom/src/event/Event.ts

```diff
@@ -35,14 +35,12 @@ export default class Event {
 	 * @param type Event type.
 	 * @param [eventInit] Event init.
 	 */
-	constructor(type: string, eventInit: IEventInit = null) {
+	constructor(type: string, eventInit: IEventInit = {}) {
```

Comment:
Wow, good catch!

I can't understand/verify from the spec if `new Event('click', **null**)` is legit (of course it is since Chrome & Firefox allow it). Can you point the line that says that?

https://dom.spec.whatwg.org/#interface-event
> constructor(DOMString type, **optional** EventInit eventInitDict = **{}**)

The `EventInit eventInitDict = **{}**` part is misleading. I guess it's `optional` that means `undefined | null` and not just `undefined`

So I've added a commit that does what you ask:
> eventInit: IEventInit | null = null
> this.height = eventInit?.height ?? 1

Btw I strongly suggest you to enable [`strict` TypeScript mode](https://www.typescriptlang.org/tsconfig#strict) which turns on [`strictNullChecks`](https://www.typescriptlang.org/tsconfig#strictNullChecks).

Reply:
I agree that `strictNullChecks` should be enabled.

I think that optional values in general in the spec can be null or undefined, but I'm not sure. I just tried in FF and Chrome and I assumed that they are following it correctly.